cli/cmd/install.go

```diff
@@ -152,9 +153,37 @@ odigos install --kubeconfig <path-to-kubeconfig>
 
 # Install Odigos onprem tier for enterprise users
 odigos install --onprem-token ${ODIGOS_TOKEN} --profile ${YOUR_ENTERPRISE_PROFILE_NAME}
+
+# Install Odigos and connect the cluster to forward data to the centralized backend
+odigos install --cluster-name my-cluster --central-backend-url https://central.odigos.local
```

Comment:
prefer not to use `https://central.odigos.local` as it might be confusing to users.

```suggestion
odigos install --cluster-name my-cluster --central-backend-url {CENTERAL_BACKEND_URL}
```